### DISCUSSION TITLE
env_spacing = 0 leads to an error

### DISCUSSION THREAD

**Discussion by dhruvkm2402** [Toni-SM/skrl#56] [General]
Hello,
For using Omniverse ISAAC Gym for parallel RL I want to spawn all the robots at one location, I try to set env_Spacing=0 but it leads to an error. How do I change that? If I do env_Spacing = 0.5 as an example, it works.

**Comment by Toni-SM** [comment]
Hi @dhruvkm2402 

The problem could be in the way Omniverse Isaac Gym filters collisions between environments so that they are ignored. 
The following post may provide some guidance on the issue: https://github.com/NVIDIA-Omniverse/OmniIsaacGymEnvs/issues/3

**Reply by dhruvkm2402** [reply]
Hi Toni,
yes, the error is related to gpu buffers. I increased the size of each, referred to the anymal example and it worked.
